www_src/pages/element/link-editor.jsx

```diff
@@ -28,8 +27,18 @@ var LinkEditor = React.createClass({
     }
 
   },
-  onChangeLinkClick: function () {
-    this.refs.notImplementedWarning.show();
+  onDestClick: function () {
+    var metadata = {
+      elementID: this.props.params.element,
+      pageID: this.props.params.page
```

Comment:
i think you need to pass `projectId` as well -- right now, it doesn't matter since links only link to the same page, but in the future this could cause bugs